.Jenkinsfile

```diff
@@ -32,20 +32,24 @@ pipeline {
                     script: './bin/console cache:warmup --env=test'
                 )
                 sh(
-                    label: 'Running static code analysis - CodeSniffer',
-                    script: './vendor/bin/phpcs -p'
+                    label: 'Running test suite - PHPUnit',
+                    script: 'source .env.test && XDEBUG_MODE=coverage ./bin/phpunit --coverage-xml=var/log/phpunit/coverage/coverage-xml --coverage-clover=var/log/phpunit/coverage.xml --log-junit=var/log/phpunit/coverage/junit.xml'
                 )
                 sh(
-                    label: 'Running static code analysis - Mess Detector',
-                    script: './vendor/bin/phpmd src,tests json phpmd.xml'
+                    label: 'Checking test coverage - PHPUnit',
+                    script: './bin/coverage-checker var/log/phpunit/coverage.xml 100'
                 )
                 sh(
-                    label: 'Running testing suite - PHPUnit & Infection',
-                    script: 'XDEBUG_MODE=coverage && ./vendor/bin/infection --threads=$(nproc) --min-msi=99 --test-framework-options="--coverage-clover=var/log/phpunit/coverage.xml"'
+                    label: 'Running mutation testing - Infection',
+                    script: 'source .env.test && ./vendor/bin/infection --threads=$(nproc) --min-msi=99 --no-progress --show-mutations --skip-initial-tests --coverage=var/log/phpunit/coverage'
                 )
                 sh(
-                    label: 'Checking test coverage - PHPUnit',
-                    script: './bin/coverage-checker var/log/phpunit/coverage.xml 100'
+                    label: 'Running static code analysis - CodeSniffer',
+                    script: './vendor/bin/phpcs -p'
+                )
+                sh(
+                    label: 'Running static code analysis - Mess Detector',
+                    script: './vendor/bin/phpmd src,tests json phpmd.xml'
```

Comment:
Though we were moving the static analysis to the start, or it didn't worked? Or not now? :)